stories/examples/zoom.stories.tsx

```diff
@@ -240,8 +212,8 @@ export function RectangleZoom() {
             />
           )}
         </Annotations>
-        <Axis min={minX} max={maxX} id="x" position="bottom" label="PC 1" />
-        <Axis min={minY} max={maxY} id="y" position="left" label="PC 2" />
+        <Axis min={0} max={50} position="bottom" label="PC 1" />
+        <Axis min={0} max={50} position="left" label="PC 2" />
```

Comment:
Note that these are the default min/max values. If the parent plot controller contains values, they take precedence.
Priority is Plot controller > Axis > Series.